app/gui/src/ide/integration.rs

```diff
@@ -73,7 +73,7 @@ impl Model {
         }
     }
 
-    /// Open a project through project manager API.
+    /// Open a project by name.
```

Comment:
Please add an information that it makes two calls to the Project Manager (first for listing projects, the second for opening one). This affects performance, and this might be important for a potential future caller of this function.

Reply:
Done